ui-core/src/participant/landing/sections/FrequentlyAskedQuestionsTemplate.tsx

```diff
@@ -7,12 +7,12 @@ import _ from 'lodash'
 import React, { useCallback, useEffect, useRef, useState } from 'react'
 
 import { SectionConfig } from '../../../types/landingPageConfig'
-import { getSectionStyle } from '../../util/styleUtils'
+import { applyAllowedStyles, getSectionStyle } from '../../util/styleUtils'
```

Comment:
```suggestion
import { applyAllowedStyles, getSectionStyle } from 'util/styleUtils'
```

Reply:
All of these import tweaks require some changes to the tsconfig for ui-core (which explains why we've got so many verbose imports in ui-core). Since I think a bunch of imports and config is going to change anyway with Vite, mind if I defer?